### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by Grajo**
This is to replace current travis.yml

### CODE REVIEW

**Comment by Grajo** [comment]
Hey I think this is all  we need to **migrate** from **travis** to **circle ci!** However, it still requires not to expect the `continuous-integration/travis-ci` to be executed anymore. I bet this is some setting in the repo that I have currently no access to, due to lack of privileges. I think it will be good practice to add to the `php CI/build` the **Required** tag too, but again this probably needs to be done through repository settings. What do you think @AdamWhittingham ?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: php CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        php-version: ['7.2']
```
Comment (AdamWhittingham) [inline .github/workflows/ci.yml:10]: I think this key is wrong:
```suggestion
        php_version: ['7.2']
```

**Comment by AdamWhittingham** [comment]
@Grajo Ah, I think you're right and it's a repo level setting to enforce the required checks (if it were a config file in the repo then someone could change it and release without the required check!) - we'll need some help in disabling it.
Otherwise, it looks lie the new builds are failing because the key for PHP version isn't quite right (needs an underscore rather than a `-`)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        os: [ubuntu-latest, macos-latest, windows-latest]
```
Comment (AdamWhittingham) [inline .github/workflows/ci.yml:11]: Great idea to test more OS's 👍

**Review by AdamWhittingham — COMMENTED** [review]
Changing `php-version` to `php_version` has fixed the Linux tests but looks like Windows has a path issue we need to work through

**Review by Nevett — CHANGES_REQUESTED** [review]
Take a look at running tests/linting as part of this also

**Review by Nevett — APPROVED** [review]

**Pull request merged**